nixos/modules/services/misc/asusd.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.asusd;
+  tomlFormat = pkgs.formats.toml {};
+in
+{
+  options = {
+    services.asusd = {
+      enable = mkEnableOption ''
+        Provides an interface for rootless control of some system functions most users wish to control such as fan speeds, keyboard LEDs, graphics modes.
```

Comment:
Please take a quick peek at the definition of `mkEnableOption` for desired string here.

Reply:
Ah indeed. Thanks for the heads-up. Updated the description to be more fitting.

Reply:
Sorry you still need to drop the trailing period! Sometimes `mkEnableOption` is more confusing than the value it provides...

Reply:
Oh oof, keen eye. That is indeed confusing. I've removed the period at the end of the string, kept the one at the end of the first line.

> Sometimes mkEnableOption is more confusing than the value it provides...

Indeed :sweat_smile: